packages/create-app/templates/default-app/packages/backend/package.json.hbs

```diff
@@ -27,7 +27,7 @@
     "@backstage/plugin-proxy-backend": "^{{version '@backstage/plugin-proxy-backend'}}",
     "@backstage/plugin-scaffolder-backend": "^{{version '@backstage/plugin-scaffolder-backend'}}",
     "@backstage/plugin-techdocs-backend": "^{{version '@backstage/plugin-techdocs-backend'}}",
-    "@octokit/rest": "^18.0.0",
+    "@octokit/rest": "18.0.6",
```

Comment:
@Rugvip, similar question to https://github.com/backstage/backstage/pull/3583#discussion_r539029858 just this time related to this commit ( ffa941bb0c2cfc1a5e3fc9fd55ceaf8e3e436636 ) and the resulting TypeScript / yarn.lock fun.